lib/arxiv/string_scrubber.rb

```diff
@@ -1,7 +1,23 @@
 module Arxiv
   class StringScrubber
-     def self.scrub(string)
+    def self.scrub(string)
       string.gsub("\n", ' ').strip.squeeze(" ")
```

Comment:
The `g` in `gsub` stands for global (I think). It will scan through a string and replace all instances of a given pattern / string. `sub` does the same thing but only replaces the first instance (i.e. not global)